src/api/BdmsContextExtensions.cs

```diff
@@ -0,0 +1,360 @@
+﻿using BDMS.Models;
+using Bogus;
+using Microsoft.EntityFrameworkCore;
+using NetTopologySuite.Geometries;
+using System.Globalization;
+
+namespace BDMS;
+
+#pragma warning disable CA1505
+/// <summary>
+/// The EF database context containing data for the BDMS application.
+/// </summary>
+public static class BdmsContextExtensions
+{
+    /// <summary>
+    /// Seed data for <see cref="Workgroup"/>, <see cref="Borehole"/> and <see cref="Stratigraphy"/>.
+    /// </summary>
+    public static void SeedData(this BdmsContext context)
+    {
+        using var transaction = context.Database.BeginTransaction();
+
+        // Set Bogus Data System Clock
+        Bogus.DataSets.Date.SystemClock = () => DateTime.Parse("01.01.2022 00:00:00", new CultureInfo("de_CH", false));
+
+        // Seed Workgroups
+        var workgroup_ids = 2;
+        var workgroupRange = Enumerable.Range(workgroup_ids, 5);
+        var fakeWorkgroups = new Faker<Workgroup>()
+           .StrictMode(true)
+           .RuleFor(o => o.Id, f => workgroup_ids++)
+           .RuleFor(o => o.Name, f => f.Music.Genre())
+           .RuleFor(o => o.Created, f => f.Date.Past().OrNull(f, .1f))
+           .RuleFor(o => o.Disabled, f => f.Date.Past().OrNull(f, .1f))
+           .RuleFor(o => o.IsSupplier, f => f.Random.Bool().OrNull(f, .1f))
+           .RuleFor(o => o.Settings, f => null)
+           .RuleFor(o => o.Boreholes, _ => default!);
+        Workgroup SeededWorkgroups(int seed) => fakeWorkgroups.UseSeed(seed).Generate();
+        context.Workgroups.AddRange(workgroupRange.Select(SeededWorkgroups));
+        context.SaveChanges();
+
+        // ranges for exsiting tables
+        var userRange = Enumerable.Range(1, 5);
+
+        var cantonRange = Enumerable.Range(1, 51);
+        var municipalitieRange = Enumerable.Range(1, 2371);
```

Comment:
Q: Spricht was dagegen, die auch gleich zu seeden?

Reply:
Die gibts ja schon als SQL scripts die ohnehin ausgeführt werden. Fändet ihr es besser wenn wir die statt dessen seeden? Und brauchts die scripts nicht ohnehin auch für prod dann?

Reply:
Die SQL-Skripts fallen demnächst weg und werden nur lokal und nicht für Prod verwendet. Wir können entweder jetzt schon die Kantone und Gemeinden seeden oder warten ab, bis die SQL-Skripts wegfallen. 

@MiraGeowerkstatt Was meinst du, wollen wir die Daten gleich jetzt seeden? Sind gerade sowieso in der Thematik drin.

Reply:
Ja klar, dann können wir die auch gleich seeden.

Reply:
Falls du die `User` auch gleich angehst, wärs wahrscheinlich am einfachsten das mit SQL zu machen... wenn das aus dem Seeder heraus irgendwie geht. 🤷🏼  Die Problematik ist da der Password Hash der mit einer PostgreSQL Extension (resp. `crypt` Funktion) erstellt wird. Ein Beispiel SQL Script findest du [hier](https://github.com/geoadmin/suite-bdms/blob/main/config/create-user.sql).